# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1404204513 14400
#      Tue Jul 01 04:48:33 2014 -0400
# Branch stable
# Node ID 46df2c7b8f4dec400ce353370f2e47a02926026d
# Parent  23681c9ea7baeaec05e889e39f8c77ea94139030
3.8.2 release candidate 2

* configure.ac (OCTAVE_VERSION): Bump to 3.8.2-rc1.
(OCTAVE_MINOR_VERSION): Bump to 2-rc2.
(OCTAVE_RELEASE_DATE): Set to 2014-07-01.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -14,31 +14,31 @@ dnl Copyright (C) 1993-2014 John W. Eato
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ### 
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
 AC_PREREQ([2.62])
-AC_INIT([GNU Octave], [3.8.2-rc1], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [3.8.2-rc2], [http://octave.org/bugs.html], [octave])
 
 dnl Note that the version number is duplicated here and in AC_INIT
 dnl because AC_INIT requires it to be static, not computed from
 dnl shell variables.
 OCTAVE_MAJOR_VERSION=3
 OCTAVE_MINOR_VERSION=8
-OCTAVE_PATCH_VERSION=2-rc1
+OCTAVE_PATCH_VERSION=2-rc2
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2014 John W. Eaton and others."
 
-OCTAVE_RELEASE_DATE="2014-06-06"
+OCTAVE_RELEASE_DATE="2014-07-01"
 
 ## The "API version" is used as a way of checking that interfaces in the
 ## liboctave and libinterp libraries haven't changed in a backwardly
 ## incompatible way when loading .oct files.  A better way to do that is
 ## with library versioning, but not all systems support that.
 ## NOTE: This macro will be removed in a future version of Octave.  If
 ## you insist on checking for features using a version number, use the
 ## OCTAVE_MAJOR_VERSION, OCTAVE_MINOR_VERSION, and
